src/templates/app/user/authenticate.jsx

```diff
@@ -350,6 +409,65 @@ const Authenticate = () => (
                             </div>
                         </div>
                     </TabContent>
+                    <TabContent id='edd'>
+                        {/* proof of income content */}
+                        <div id='not_authenticated_edd' className='invisible'>
+                            <EddMessage />
+                        </div>
+
+                        <div id='verified_edd' className='center-text gr-gutter gr-padding-20 invisible'>
```

Comment:
if this is proof of income content, why we see here a lot of mentions about other features ?